package.json

```diff
@@ -59,14 +58,22 @@
     "ldapjs": "1.0.1"
   },
   "devDependencies": {
+    "babel-core": "6.21.0",
+    "babel-loader": "6.2.10",
+    "babel-preset-es2015": "6.18.0",
     "chai": "3.5.0",
     "eslint": "3.12.2",
     "font-awesome": "4.7.0",
     "handlebars": "4.0.6",
+    "handlebars-loader": "1.4.0",
     "istanbul": "0.4.5",
+    "jquery": "2.1.1",
+    "jquery-ui": "1.12.1",
```

Comment:
There is a massive [diff between 1.11.1 (version in currently in repo) and 1.12.1](https://github.com/jquery/jquery-ui/compare/1.11.1...1.12.1). I tried and we can't use the tag directly from GitHub so I guess we'll have to go with that.
I don't think we use a lot from jQuery UI anyway, this can probably go at some point...